### ISSUE TITLE
The loss curves shows that training of the model does not converge

### ISSUE THREAD

**Issue by ghost** [yihongXU/deepMOT#40]
Hi,
I am following your great work recently.
I download the pre-trained DHN model weights and train a tracktor. 
I checked the log in `result_full` and got oscillating loss as the following:
![image](https://user-images.githubusercontent.com/26536908/96120989-f8f3d980-0f21-11eb-952e-4a6243703634.png)
![image](https://user-images.githubusercontent.com/26536908/96121003-00b37e00-0f22-11eb-849b-f035bdf73057.png)

**Comment by ghost** [comment]
I have got correct evaluation results. 
I just wonder why the loss curve looks like that.
